unittest/c-library/CMakeLists.txt

```diff
@@ -1,26 +1,26 @@
 
 add_executable(test_library_open test_library_open.cpp test_main.cpp)
 target_link_libraries(test_library_open PRIVATE lammps GTest::GMock)
-add_test(LibraryOpen test_library_open)
+add_test(NAME LibraryOpen COMMAND test_library_open)
```

Comment:
Minor nitpick: without any extra arguments these CMake changes are purely cosmetic now.

Reply:
No they are not. Without them the test executables are not found (at least on windows) with a multi-config builder. I found that out empirically when comparing tests that would find them to those that would not.

Reply:
That is interesting! I didn't know that!

Maybe this is part of the transformations: "Unlike the above NAME signature no transformation is performed on the command-line to support target names or generator expressions." (from https://cmake.org/cmake/help/latest/command/add_test.html)

Who knows, the new syntax is anyway more descriptive and we should use it everywhere.